### PR TITLE
Improve Code Coverage

### PR DESCRIPTION
**Pull request by munir-rakuten**
# Description
Increase code coverage for better quality of code.

## Links

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/RetrofitCreatorUtils.kt [hunk 1]
```diff
@@ -64,19 +64,21 @@ private fun provideHeaderInterceptor(): Interceptor = Interceptor { chain ->
...
 }
 
-private fun extractBaseUrl(baseUrl: String): String {
+@VisibleForTesting
+internal fun extractBaseUrl(baseUrl: String): String {
     return try {
         val url = URI.create(baseUrl).toURL()
         url.authority
-    } catch (e: MalformedURLException) {
+    } catch (e: IllegalArgumentException) {
```
Comment (khairul-alam-licon) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/RetrofitCreatorUtils.kt:81]: Do you think, we can catch both? 🤔

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 🤔

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/RetrofitCreatorUtils.kt (hunk 1 above)
Comment (munir-rakuten) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/RetrofitCreatorUtils.kt:83]: @khairul-alam-licon yes we can, updated it!! thanks 👍

**Review by corycaywood — APPROVED** [review]
Looks good! 👍

**Pull request merged**